security/certificate_types_descriptions/olm-certificates.adoc

```diff
@@ -13,3 +13,17 @@ All certificates for Operator Lifecycle Manager (OLM) components (`olm-operator`
 When installing Operators that include webhooks or API services in their `ClusterServiceVersion` (CSV) object, OLM creates and rotates the certificates for these resources. Certificates for resources in the `openshift-operator-lifecycle-manager` namespace are managed by OLM.
 
 OLM will not update the certificates of Operators that it manages in proxy environments. These certificates must be managed by the user using the subscription config.
+
+[role="_additional-resources"]
+.Next steps
+
+* xref:../../operators/admin/olm-configuring-proxy-support.adoc#olm-inject-custom-ca_olm-configuring-proxy-support[Overriding proxy settings of an Operator]
```

Comment:
I think this xref points to the section right below "Overriding proxy settings..."